css/events.css

```diff
@@ -0,0 +1,232 @@
+/*
+===================================
+===================================
+Styles for an individual page
+
+- Rename this file to match the page you are working on (e.g. home.css, about.css, etc.)
+- Create a new file for your page and link this file to it in the <head> of your HTML document
+- Variables from global.css will work in this CSS file as well. Do not add any variables to this file.
+===================================
+===================================
+*/
+
+/* 
+===================================
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+
+/* Cards */
+
+.card-container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+	gap: var(--spacing-sm);
+	padding: var(--spacing-med);
+}
+
+.events-container {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
```

Comment:
Nice and DRY! I am noticing a few breakpoints could be added in to accommodate for the events wrapping in some strange ways. I think making some adjustments on flex layout or images for the following viewport widths might help:

- 1507px --> 1080px
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/2ecabb6a-b701-4ff7-8200-f2fd0613235d)

- 1070px --> 1024px
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/bdd23326-e4ae-4a22-b6e3-5e7f699f8936)

- 821px - 768px things seem to be on the left side
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/370dd57d-e551-41fe-a76b-871a26910ed1)

Reply:
Thanks for checking these sizes, will fix!

Reply:
Updated, please test again @SolivanLau

Reply:
I think this looks good! nothing is shifting to left or right sides from my end!